src/test/librbd/test_WriteLogMap.cc

```diff
@@ -0,0 +1,318 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#include "test/librbd/test_fixture.h"
+#include "test/librbd/test_support.h"
+#include "librbd/cache/ReplicatedWriteLog.h"
+
+void register_test_write_log_map() {
+}
+
+namespace librbd {
+namespace cache {
+namespace rwl {
+
+struct TestLogEntry {
+  uint64_t image_offset_bytes;
+  uint64_t write_bytes;
+  uint32_t referring_map_entries = 0;
+  TestLogEntry(const uint64_t image_offset_bytes, const uint64_t write_bytes)
+    : image_offset_bytes(image_offset_bytes), write_bytes(write_bytes) {
+  }
+  const uint64_t get_offset_bytes() { return image_offset_bytes; }
```

Comment:
GCC would split warning messages at seeing this. could you instead move `const` after `()`?

Reply:
OK